eng/Versions.props

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <PreReleaseVersionLabel>prerelease</PreReleaseVersionLabel>
```

Comment:
This is a rather unusual prerelease label. Usually we use alpha or preview instead.

Reply:
Let's just use `preview1`. I think we won't be ready to call it "shipped" until Feb anyway.